### PR TITLE
Adds instructions for GitHub workflow

### PR DESCRIPTION
**Pull request by edieblu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,54 @@ You may change this to 'postgres://username:password@localhost:5432/girlscode_li
...
+From there you can create a pull request. The pull request will compare the changes you made to the existin code. Someone will review the code and then add it to the (upstream) master.
+
+## Updating your remote
+
+The upstream remote will update with every merged commit (this can be from you, or other contributors).
+
+Using the setup suggested above, this is how you can keep your remote up-to-date:
+
+1. To fetch all the changes, type: `git fetch upstream`
+2. To incorporate those changes to your local master type: `git merge --ff-only upstream/master`
```
Comment (singloon) [inline README.md]: I wonder if we should have as section on gitconfig e.g.  git config --add branch.master.mergeoptions --ff-only if we're following this model

Comment (edieblu) [reply README.md]: Or maybe suggest adding this to the global gitconfig file? 
```
[branch "master"]
mergeoptions = --ff-only
```

Comment (singloon) [reply README.md]: I'm not sure if it should be --global. As they may have different projects with different git flows.

Comment (NeilNjae) [reply README.md]: At this stage, I think keep things simpler and explicit, rather than hiding things inside options.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,54 @@ You may change this to 'postgres://username:password@localhost:5432/girlscode_li
...
+## Remote repository setup
+
+We suggest using two remote repositories:
+- one called upstream, in our case `https://github.com/GirlsCodeMK/libraryapp-hackathon-js-starthere`
+- one called origin, which is your fork `https://github.com/<YOUR GITHUB USERNAME>/libraryapp-hackathon-js-starthere`
+
+You can check your remote(s) by running the following command in your command line / terminal:
+`git remote -v`
+
+You can add a remote by typing: `git add remote origin https://github.com/<YOUR GITHUB USERNAME>/libraryapp-hackathon-js-starthere`
```
Comment (NeilNjae) [inline README.md]: Change this to 
`git add remote upstream https://github.com/GirlsCodeMK/libraryapp-hackathon-js-starthere`
(the repo will already have an `origin` remote).

File: README.md [hunk 3]
```diff
@@ -28,6 +28,54 @@ You may change this to 'postgres://username:password@localhost:5432/girlscode_li
...
+## Updating your remote
+
+The upstream remote will update with every merged commit (this can be from you, or other contributors).
+
+Using the setup suggested above, this is how you can keep your remote up-to-date:
+
+1. To fetch all the changes, type: `git fetch upstream`
+2. To incorporate those changes to your local master type: `git merge --ff-only upstream/master`
+Now your local master is up-to-date. To make sure your origin (your fork) is up-to-date too, type:
+3. `git push -f origin master`
```
Comment (NeilNjae) [inline README.md]: I wouldn't include the `-f` option here. If you need to force the push, something's gone wrong, and the person should try to understand what that wrong thing is.

**Pull request merged**